tests/test_display_expression.cpp

```diff
@@ -0,0 +1,71 @@
+#include "../src/combinator.hpp"
+#include <iostream>
+#include <metricq/json.hpp>
+
+struct TestDisplayExpression
+{
+    nlohmann::json input;
+    std::string expected;
+};
+
+int main()
+{
+    try
+    {
+        std::vector<TestDisplayExpression> testCases;
+
+        testCases.emplace_back(TestDisplayExpression{
+            { { "expression",
+                { { "operation", "*" },
+                  { "left", 5 },
+                  { "right", { { "operation", "-" }, { "left", 45 }, { "right", 3 } } } } } },
+            "(5 * (45 - 3))" });
```

Comment:
```suggestion
            "5 * (45 - 3)" });
```

While it is technically correct, there should not be these out-most parenthesis.